### PR TITLE
6.6.3: Add etcd-client-log-level backend config flag

### LINKED ISSUE #3552: Document new --etcd-client-log-level flag / config option
### Description
We're adding a new `--etcd-client-log-level` flag in the 6.6.3 release which controls the log level of the internal Etcd client used within sensu-backend. We've also updated the description of the `--etcd-log-level`  flag to specify that it controls the Etcd server log level.

The output of `sensu-backend start --help` will show:
```
--etcd-client-log-level string               etcd client logging level [panic, fatal, error, warn, info, debug] (default "error")
--etcd-log-level string                          etcd server logging level [panic, fatal, error, warn, info, debug]
```

The default Etcd client log level is `error`.

Implemented in PR: https://github.com/sensu/sensu-go/pull/4560

### Category
This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/backend/

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
In the backend reference:
- Adds the etcd-client-log-level to the list of config flags
- Adds a description table for etcd-client-log-level
- Adds "server" to etcd-log-level sensu-backend start --help response in list of config flags

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/3552

### CODE REVIEW

File: content/sensu-go/6.6/observability-pipeline/observe-schedule/backend.md [hunk 1]
```diff
@@ -1074,6 +1075,21 @@ sensu-backend start --etcd-client-cert-auth{{< /code >}}
 /etc/sensu/backend.yml example | {{< code shell >}}
 etcd-client-cert-auth: true{{< /code >}}
 
+<a id="etcd-log-level"></a>
+
+| etcd-client-log-level  |      |
+-------------|------
+description  | Logging level for the embedded etcd client: `panic`, `fatal`, `error`, `warn`, `info`, or `debug`. {{% notice note %}}
```
Comment (amdprophet) [inline content/sensu-go/6.6/observability-pipeline/observe-schedule/backend.md]: Embedded is an overloaded term. One may assume that embedded etcd client only applies to the etcd client when embedded etcd is in use. This flag is used for the etcd client when used with embedded or external etcd.

Perhaps "internal" is more appropriate?

Comment (hillaryfraley) [reply content/sensu-go/6.6/observability-pipeline/observe-schedule/backend.md]: Yes, I can change it to "internal"

A related question...is "embedded" appropriate for the etcd-log-level flag? We say "embedded" in that description table (see https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/backend/#etcd-log-level) but I can change that to "internal" too if "embedded" is incorrect.

**Review by amdprophet — APPROVED** [review]

File: content/sensu-go/6.6/observability-pipeline/observe-schedule/backend.md (hunk 1 above)
Comment (amdprophet) [reply content/sensu-go/6.6/observability-pipeline/observe-schedule/backend.md]: @hillaryfraley yes, embedded is appropriate for the non-client etcd log level. We can only control the etcd server log level if it's running embedded within sensu-backend. External etcd servers are a separate process thus we have no control.

Comment (hillaryfraley) [reply content/sensu-go/6.6/observability-pipeline/observe-schedule/backend.md]: Roger that, thank you @amdprophet !

**Pull request merged**